source/exportdialog.cpp

```diff
@@ -3,6 +3,7 @@
 #include <QFileDialog>
 #include <QMessageBox>
 
+#include <algorithm>
```

Comment:
I think the correct style is
```suggestion
#include <algorithm>

```